python/ray/data/read_api.py

```diff
@@ -562,6 +562,15 @@ def read_csv(
         >>> ray.data.read_csv( # doctest: +SKIP
         ...     ["s3://bucket/path1", "s3://bucket/path2"])
 
+        >>> # Use different delimiter. The partition_filter=None is needed here
+        >>> # because by default read_csv only reads .csv files.
+        >>> from pyarrow import csv
+        >>> parse_options = csv.ParseOptions(delimiter="\\t")
+        >>> ray.data.read_csv( # doctest: +SKIP
```

Comment:
This needs to be skipped at the moment which is very very sad -- tracked in https://github.com/ray-project/ray/issues/27853